Content.Client/Interface/Controls/ControlLabel.cs

```diff
@@ -0,0 +1,28 @@
+using Content.Shared.Interface;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+
+namespace Content.Client.Interface.Controls
+{
+    class ControlLabel : InterfaceControl {
+        private Label _label;
+
+        public ControlLabel(ControlDescriptor controlDescriptor, ControlWindow window) : base(controlDescriptor, window) { }
+
+        protected override Control CreateUIElement() {
+            _label = new Label() {
+                HorizontalAlignment = Control.HAlignment.Stretch,
+                VerticalAlignment = Control.VAlignment.Stretch,
```

Comment:
Why set this to `Stretch` instead of `Center`?

Reply:
Because I copypasted it from somewhere